net/BUILD.gn

```diff
@@ -1405,7 +1405,7 @@ component("net") {
     sources += [ "disk_cache/blockfile/mapped_file_posix.cc" ]
   }
 
-  if (false) {
+  if (is_android) {
```

Comment:
I thought this is not true for us and we'd use is_starboard?

Reply:
I'm not sure why, but this change does make the linking issue go away?

Reply:
Upstream: https://source.chromium.org/chromium/chromium/src/+/main:net/BUILD.gn;l=1318-1323?q=net%2FBUILD.gn&ss=chromium%2Fchromium%2Fsrc. Looks to me like we should bring back that conditional and add `|| is_starboard` and rely on the NOTIMPLEMENTED here: https://source.chromium.org/chromium/chromium/src/+/main:net/base/network_change_notifier_passive.cc;l=122

Reply:
We want to add `|| is_starboard` as there should not be platform differences in the cobalt layer of the build, i.e. windows should use these same functions to link when it is built